LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -244,9 +244,9 @@ extension TherapySettingsView {
         section(for: .insulinModel) {
             if self.viewModel.therapySettings.insulinModel != nil {
                 VStack(alignment: .leading, spacing: 4) {
-                    Text(InsulinModelSettings(from: self.viewModel.therapySettings.insulinModel!).title)
```

Comment:
Is `InsulinModelSettings.init(from:)` still used elsewhere, or can that initializer be deleted now?

Reply:
`StoredSettings` is still using it in some places, so I'll leave the initializer in there